src/wp-includes/class-wp-block-type-registry.php

```diff
@@ -142,6 +166,10 @@ public function get_registered( $name ) {
 			return null;
 		}
 
+		if ( isset( $this->registered_block_type_aliases[ $name ] ) ) {
```

Comment:
Thought: Separate methods for checking if a block is registered vs an alias is registered.